test/test_torch.py

```diff
@@ -7590,6 +7609,21 @@ def test_flip(self, device):
         a = torch.tensor([False, True])
         self.assertEqual(a.flip(0), torch.tensor([True, False]))
 
+    @dtypes(torch.cfloat, torch.cdouble)
+    @unittest.skipIf(not TEST_NUMPY, "Numpy not found")
+    def test_complex_flip(self, device, dtype):
```

Comment:
Is there an existing flip tests and/or would it make sense to extend this one to other dtypes, too?

Reply:
https://github.com/pytorch/pytorch/pull/37826#discussion_r420353275

Reply:
@anjali411 Thanks for adding the reference. Just to expand a bit more on it. The existing tests don't compare against `numpy` but against an expected result. Also they don't explicitly use the `dtype` and thus multiple changes would be needed to support taking the `dtype` argument.

Reply:
OK.